fl4health/clients/basic_client.py

```diff
@@ -24,6 +24,10 @@
 from fl4health.utils.metrics import Metric, MetricManager
 
 T = TypeVar("T")
+BatchType = TypeVar(
```

Comment:
Is this being used anywhere? I don't see it anywhere on the branch, but I may be missing something?

Reply:
Good catch. This is from a previous version and is not currently used anywhere. I'll remove it.